Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs

```diff
@@ -99,7 +101,8 @@ public void AddItemSlot(EntityUid uid, string id, ItemSlot slot, ItemSlotsCompon
             if (itemSlots.Slots.TryGetValue(id, out var existing))
             {
                 if (existing.Local)
-                    Log.Error($"Duplicate item slot key. Entity: {EntityManager.GetComponent<MetaDataComponent>(uid).EntityName} ({uid}), key: {id}");
```

Comment:
There are a lot of unnecessary line splits and whitespace changes here. It makes it hard to see if any of the code has changed, and it's going to log them on GitHub's 'blame' list as file changes

Reply:
yeah, sorry I accidentally formatted the file.

Reply:
The actual changes are pretty much localized to one or two functions, this is a pretty small bugfix